packages/@o3r/rules-engine/src/fact/fact.spec.ts

```diff
@@ -31,6 +37,7 @@ describe('Rules engine fact', () => {
     factsService.register();
     await jest.runAllTimersAsync();
 
+    // eslint-disable-next-line @typescript-eslint/unbound-method -- method is not called here, we use reference for the mock engine
     expect(mockEngine.upsertFacts).toHaveBeenCalledTimes(1);
```

Comment:
Is it still failing after the import of the jest-unbound-method?

Reply:
will be fixed with the rebase